playbooks/prosody_qa.yml

```diff
@@ -0,0 +1,27 @@
+- hosts: solr_staging
+  connection: ssh
+  remote_user: pulsys
+  vars:
+    app_name: cdh_solr   # referenced in deploy user bash profile
+  roles:
+    - deploy_user
+
+- hosts: prosody_qa
+  connection: ssh
+  remote_user: pulsys
+  become: true
+  roles:
+    # - create_deployment
```

Comment:
We're ok leaving these out now still?

Reply:
good catch! let's leave commented out until we get the deploy working more but make sure to uncomment before we merge